### ISSUE TITLE
Improve color contrast in tree-node

### ISSUE THREAD

**Issue by magicp-github** [devtools-html/debugger.html#1940]
Existing color contrast of tree-node (e.g. arrow, domain svg...) is difficult to see in the light and dark theme. It should be improved.

![improve-color-contrast-in-node-tree](https://cloud.githubusercontent.com/assets/13914314/22631161/585974ce-ec4a-11e6-8da8-3acb22626d2b.png)

**Comment by jasonLaster** [comment]
@clarkbw thoughts?

**Comment by clarkbw** [comment]
When that was the clickable target I agreed with this.  But I'm not sure anymore. I don't know that we want to make the arrows on equal weight as the text.  Perhaps if we went with lighter / thinner arrows then I'd feel better about it.

**Comment by jasonLaster** [comment]
That makes sense.
I'd also like to discuss adopting Chrome's color pattern for different types of nodes. Directory, source, source map directory.
